package.json

```diff
@@ -54,7 +54,7 @@
     "prettier": "2.2.1",
     "prettier-plugin-solidity": "1.0.0-beta.10",
     "pretty-quick": "^2.0.1",
-    "secp256k1": "^3.7.1",
+    "secp256k1": "^5.0.1",
```

Comment:
for posterity, bumping this because of https://github.com/advisories/GHSA-584q-6j8j-r5pm